tests/test_datastructures.py

```diff
@@ -43,3 +49,41 @@ def test_for_grouper(self):
             transform=lambda x: x.pk,
             ordered=False
         )
+
+    def test_grouper_model(self):
+        versionable = VersionableItem(content_model=PollContent, grouper_field_name='poll')
+
+        self.assertEqual(versionable.grouper_model, Poll)
+
+
+class VersionableItemProxyModelTestCase(CMSTestCase):
```

Comment:
Now that you have all these tearDown methods, do you still need the person app?

Reply:
Yep, otherwise we still have the problem with creating proxies that have already been registered during project startup.